operator/internal/controller/redpanda/redpanda_decommission_controller.go

```diff
@@ -360,23 +368,36 @@ func (r *DecommissionReconciler) reconcileDecommission(ctx context.Context, log
 			nodesDownMap.Add(node)
 		}
 
+		bl, err := adminAPI.Brokers(ctx)
+		if err != nil {
+			return ctrl.Result{}, fmt.Errorf("could not get brokers: %w", err)
+		}
+
+		addresses := map[string]any{}
```

Comment:
super nit: _strictly_ speaking, I think declaring this map as `map[T]any{}` defeats the purpose of using a `map[T]struct{}` as a set instead of `map[T]bool`. A `struct{}` takes up less memory than `bool`, which is the argument for using it as the key. IIUC, `any` or `interface{}` will take up more memory than both as it has to interfaces have to allocate additional memory for the concrete type information of what's being stored.

Aside from that, if you know the type being stored in the map, using `any` will be confusing for future maintainers.